examples/ota-requestor-app/efr32/args.gni

```diff
@@ -23,6 +23,9 @@ pw_assert_BACKEND = "$dir_pw_assert_log"
 chip_enable_openthread = true
 chip_openthread_ftd = false
 
+# Disable stack lock tracking
```

Comment:
Please document why this is being disabled.

And if it's not going to be maintained, why not just remove it instead of leaving a broken example?